src/ducks/SANCharts/data.js

```diff
@@ -398,8 +400,34 @@ export const Metrics = {
     node: 'line',
     Component: Line,
     label: 'Twitter',
-    // dataKey: 'followersCount',
-    description: `Shows the number of followers on the project's official Twitter account over time`
+    description: `Shows the number of followers on the project's official Twitter account over time`,
+    subMetrics: SOCIAL_TWITTER_INTERVALS.map(interval => {
+      return {
+        queryKey: 'twitter_followers',
+        key: `twitter_followers_${interval}`,
+        category: 'Social',
+        node: 'line',
+        Component: Line,
+        label: `Twitter Changes ${interval}`,
```

Comment:
Timebound metric's interval is written in parenthesis. Better to keep consistency and label it as a `Twitter Changes (${interval})`